rep-2000.rst

```diff
@@ -709,6 +709,121 @@ Package manager use for dependencies:
 - OpenEmbedded: opkg
 
 
+Build System Support:
+
+- ament_cmake
+- cmake
+- setuptools
+
+Galactic Geochelone (May 2021 - May 2022)
+----------------------------------
+
+Targeted platforms:
+
++--------------+-----------------------+-------------------------+----------------------+--------------------+----------------+
+| Architecture | Ubuntu Focal (20.04)  | MacOS Mojave (10.14)    | Windows 10 (VS2019)  | Debian Buster (10) | OpenEmbedded / |
```

Comment:
```suggestion
| Architecture | Ubuntu Focal (20.04)  | MacOS Catalina (10.15)  | Windows 10 (VS2019)  | Debian Buster (10) | OpenEmbedded / |
```

We considered moving to MacOS Catalina in Foxy, but ended up on Mojave. From what I understand, Big Sur would be difficult, but has the Catalina picture improved enough to move forward for Galactic?

Reply:
Unless we get contributions from elsewhere, let's leave macOS support at Mojave.  Also, given that we currently have nobody willing to do the work to fix the current problems in macOS (from https://discourse.ros.org/t/macos-support-in-ros-2-galactic-and-beyond/17891/8), we should also mark it as Tier 3 here.